python/ray/train/gbdt_trainer.py

```diff
@@ -142,6 +144,8 @@ class GBDTTrainer(BaseTrainer):
     _tune_callback_checkpoint_cls: type
     _default_ray_params: Dict[str, Any] = {"checkpoint_frequency": 1}
     _init_model_arg_name: str
+    _num_iterations_argument: str = "num_boost_round"
```

Comment:
Is `n_estimators` also a thing people pass in?

Reply:
```suggestion
    _num_iterations_attribute: str = "num_boost_round"
```

Reply:
`n_estimators` can be passed in `params`, but lightgbm-ray/xgboost-ray will throw exceptions, forcing `num_boost_round` as the only way you can set the number of trees.